pages/careers/_open-positions.html

```diff
@@ -237,5 +320,267 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
+        one of our co-founders.
+      </li>
+      <li>
+        <strong>Work alongside customers.</strong>
+        You'll work directly with our customers to understand what problems they are facing, figure out solutions to
+        those problems, guide the team in building those solutions, share those solutions with customers, gather
+        feedback, and iterate.
+      </li>
+      <li>
+        <strong>Drive our product management practices.</strong>
+        Help improve how we build products as a company by streamlining all of our product management processes,
+        including product planning, user testing, user story mapping, task tracking, backlog grooming, sprint planning
+        and execution, market research, pricing strategy, customer communication, and more.
+      </li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>
+        <strong>Product sensibility.</strong>
+        As the future owner of several major new products at Gruntwork, we'll look to you for leadership, insights, and
+        methodologies that will help us translate our shared vision into a fantastic experience for customers.
+      </li>
+      <li>
+        <strong>Leadership.</strong>
+        We're looking for someone with a proven track record of delivering products that move the needle. You must
+        have the ability to lead a team through ideation, prototyping, user testing, development, launch, and iteration.
+      </li>
+      <li>
+        <strong>Domain expertise.</strong>
+        We don't expect you to know everything coming in, but to create products in our space, you must be able to
+        quickly build a deep understanding of our domain: DevOps, cloud software, software delivery, CI / CD, and
+        infrastructure as code. If you happen to have prior experience in software engineering, working on dev tools, o
+        other technical domains, that is a big plus.
+      </li>
+      <li>
+        <strong>Values.</strong>
+        You are inspired by <a href="/about/#our-values">our values</a>. Bonus points for a sense of humor, empathy,
+        and curiosity.
+      </li>
+    </ul>
+    <h4>Compatible time zones</h4>
+    <p>
+      You'll be working with a team in the <strong>US time zones</strong>, so you can be located in almost any
+      country as long as your time zone is no further west than Los Angeles (GMT-8) and no further east than New York
+      (GMT-4). We've found that when everyone on the team is located in similar time zones, it's easier to collaborate
+      and there's much less pressure to stay up late or get up early, so this is a hard constraint, even if you're
+      willing to work hours different from your current time zone.
+    </p>
+    <h4>Help us a create a fundamentally better DevOps experience!</h4>
+    <p>
+      If you're excited by the above:<br />
+      <!-- TODO: Fill in the proper link!!! -->
+      <a
+        href="#TODO:fill-me-in!"
```

Comment:
The job is available at https://gruntwork.workable.com/backend/jobs/2403142, though we'll need to fill in the details once this job description is done.

Reply:
Ah, thanks. I filled out the description based on this PR and published this job on our careers page.